tests/flow.md

```diff
@@ -118,3 +118,25 @@ Copying from src using `Read_source_buffer`:
 +dst: wrote "foobar"
 - : unit = ()
 ```
+
+## Pipes
+
+Writing to and reading from a pipe.
+
+```ocaml
+# Eio_main.run @@ fun env ->
+  Switch.run @@ fun sw ->
+  let r, w = Eio_unix.pipe sw in
+  Eio.Fiber.both
+    (fun () ->
+      let buf = Cstruct.create 32 in
+      let r = Eio.Flow.read r buf in
```

Comment:
In theory, this is not reliable (though it seems unlikely to fail). Should use `read_exact` here, or `Buf_read`.

Reply:
Indeed thanks! I've rebased to tidy up the history and used `read_exact`.